docs/setup/new_environment.md

```diff
@@ -83,7 +83,9 @@ This document will walk you through the process of setting up a new monolith ser
      - `ALLOWED_HOSTS`
      - `server_email`
      - `default_from_email`
-
+3. Add valid `root_email` address in public.yml file
+    - `public.yml`
+      - `root_email`
```

Comment:
```suggestion
      - `root_email`
```
I don't think this variable needs special mention - it's just like the others being edited in `public.yml`